content/04.study.md

```diff
@@ -47,6 +47,24 @@ Deep learning applied to gene expression data is still in its infancy, but the f
 Many previously untestable hypotheses can now be interrogated as deep learning enables analysis of increasing amounts of data generated by new technologies.
 For example, the effects of cellular heterogeneity on basic biology and disease etiology can now be explored by single-cell RNA-seq and high-throughput fluorescence-based imaging, techniques we discuss below that will benefit immensely from deep learning approaches.
 
+### DNA Methylation
+
+#### Inference, Imputation, and Prediction
+
+Deep learning approaches are beginning to help address some of the current limitations of feature-by-feature analysis approaches to DNA methylation data, and may help uncover additional important features necessary to understand the biological underpinnings behind different pathological states.
+One of the more popular applications is the prediction of the degree of methylation at CpG sites neighboring measured sites.
```

Comment:
Does neighboring mean immediately adjacent or in the region? How big are the windows (in rough terms)?

Reply:
It depends, I think some methods like DeepCpG predict methylation in sites of local windows, integrating local information from sequence features at distances that could be well over 1kb away from the site. Methods like DAPL could potentially be integrating information that could be more long range to impute missingness, there's no threshold on window size like DeepCpG because it's a fully connected denoising autoencoder. I think it can be an assumption that sites that are nearby could be more useful. 

The short answer is within a region of a few kb using methods like DeepCpG.